source/ios/Additions.cs

```diff
@@ -72,14 +72,17 @@ static void ConvertToNsExceptionAndAbort(object e)
 			var msg = e.ToString();
 
 			var ex = e as Exception;
-			if(ex != null) 
-				name = string.Format("{0}: {1}", ex.GetType().FullName, ex.Message);
-
+			if (ex != null) {
+				name = ex.GetType ().FullName;
+				if (ex.StackTrace != null) {
+					msg = msg.Insert (msg.IndexOf('\n'), "Xamarin Exception Stack:");
```

Comment:
as in already taken care of it or will?

the issue is being displayed like this currently: https://rink.hockeyapp.net/manage/apps/307767/app_versions/1/crash_reasons/118175066